src/lib/eina/eina_pipe.h

```diff
@@ -2,9 +2,10 @@
 #define EINA_PIPE_H
 
 # ifdef _WIN32
-#  define pipe_write(fd, buffer, size) send((fd), (char *)(buffer), size, 0)
-#  define pipe_read(fd, buffer, size)  recv((fd), (char *)(buffer), size, 0)
-#  define pipe_close(fd)               closesocket(fd)
+#  include <io.h>
+#  define pipe_write(fd, buffer, size) _write((fd), (char *)(buffer), size)
+#  define pipe_read(fd, buffer, size)  _read((fd), (char *)(buffer), size)
+#  define pipe_close(fd)               _close(fd)
```

Comment:
Could you explain why this change? `send/recv/closesocket` isn't working with something from master?

Reply:
AFAIR it wasn't, but I can't remember if this was solving the issue. But, well, I'm using the Windows equivalent here, so...should work as well.

Reply:
Ok, `send` and `recv` actually make it compile